tests/models/modules/test_masked_autoencoder.py

```diff
@@ -1,12 +1,13 @@
 import unittest
 
 import torch
-import torchvision
 
 from lightly.models import utils
 from lightly.utils import dependency
 
 if dependency.torchvision_vit_available():
+    import torchvision
```

Comment:
Moving the torchvision import inside the `if` is not really required. torchvision should always be installed.

Reply:
`torchvision_vit_available` is only `True` if the installed torchvision version includes the VisionTransformer (was added in pytorch 2.0 if I am not mistaken). Torchvision is always installed as per our requirements: https://github.com/lightly-ai/lightly/blob/master/requirements/torch.txt